packages/contracts-bedrock/contracts/universal/StandardBridge.sol

```diff
@@ -283,15 +317,48 @@ abstract contract StandardBridge {
     /**
      * @notice Finalizes an ETH bridge on this chain. Can only be triggered by the other
      *         StandardBridge contract on the remote chain.
-     *
+     * @param _localToken      Address of the sender token.
+     * @param _remoteToken     Address of the receiver token.
      * @param _from      Address of the sender.
      * @param _to        Address of the receiver.
      * @param _amount    Amount of ETH being bridged.
      * @param _extraData Extra data to be sent with the transaction. Note that the recipient will
      *                   not be triggered with this data, but it will be emitted and can be used
      *                   to identify the transaction.
      */
-    function finalizeBridgeETH(
+    function finalizeBridgeETHDeposit(
+        address _localToken,
+        address _remoteToken,
```

Comment:
According to my understand, `_localToken` must be BVM_ETH address and `_remoteToken` must be zero address. So why do we need the two parameter?